app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -438,10 +430,10 @@ private void showFeedInformation(final Feed feed, Map<String, String> alternateF
 
         title.setText(feed.getTitle());
         author.setText(feed.getAuthor());
-        description.setText(feed.getDescription());
+        description.setText(HtmlToPlainText.getPlainText(feed.getDescription()));
```

Comment:
This is no longer needed because the description is already cleaned up in beforeShowFeedInformation.